src/renderer/components/basic.lua

```diff
@@ -22,6 +22,19 @@ local function calcSizeBig(text)
   return math.ceil(calcWidth(text) / 2) * 2, math.ceil(font.height / 3) * 3
 end
 
+local function calcTextSize(node)
+  local size, root = node.styles["font-size"] and node.styles["font-size"]:match("(%d+)(r?)em")
```

Comment:
What about `0.5em`? `%d+` will only catch integers. Consider `[%d.]+`, and then `size = tonumber(size)` afterwards

Reply:
Fixed.